app/models/site.rb

```diff
@@ -0,0 +1,2 @@
+class Site < ApplicationRecord
```

Comment:
```suggestion
class RssChannel < ApplicationRecord
```

これまた無知ですいません。。。
RSSでは channel という言い方が正しいようです。。。

この辺を修正しようと思ったら結構面倒かもしれないので、時間かかりそうだったら修正しなくてOKです 👍